src/context/categories-context.jsx

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import { createContext, useContext } from "react";
+import { useState } from "react";
+import { useEffect } from "react";
```

Comment:
can have these imports in line number 2 itself

Reply:
Changes Done